backend/dal/Security/Permissions.cs

```diff
@@ -82,6 +82,18 @@ public enum Permissions
         ResearchFileEdit = 25,
 
         [Display(GroupName = "researchfile", Name = "researchfile-delete", Description = "Can delete existing research files.")]
-        ResearchFileDelete = 26
+        ResearchFileDelete = 26,
+
+        [Display(GroupName = "note", Name = "note-view", Description = "Can view notes.")]
+        NoteView = 27,
+
+        [Display(GroupName = "note", Name = "note-add", Description = "Can add new notes.")]
+        NoteAdd = 28,
+
+        [Display(GroupName = "note", Name = "note-edit", Description = "Can update existing notes.")]
+        NoteEdit = 29,
+
+        [Display(GroupName = "note", Name = "note-delete", Description = "Can delete existing notes.")]
+        NoteDelete = 30,
```

Comment:
I'm not sure how we add new claims/permissions to our system @devinleighsmith? I suspect there is more to it than just this enum?